x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/text_fields_body.tsx

```diff
@@ -43,6 +44,10 @@ export const TextFieldsBody: React.FC = () => {
           </EuiTableRowCell>
           <EuiTableRowCellCheckbox>
             <EuiCheckbox
+              aria-label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawArialLabel',
```

Comment:
```suggestion
                'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawAriaLabel',
```
Looks like just one super quick `aria` typo left!